.github/workflows/staging-build-deb.yml

```diff
@@ -649,3 +649,69 @@ jobs:
         with:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
+
+  Test-SEC-KIBANA:
+    needs: [sign-deb-artifacts]
+    runs-on: [ubuntu-16.04]
```

Comment:
Have you tried the workflow with ubuntu 18.x

Reply:
Changed to 18. It works https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/461658927